flask_common/mongo/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .documents import DocumentBase, RandomPKDocument, SoftDeleteDocument
```

Comment:
I'd prefer not to re-export it. This package doesn't result in a better or higher-level API then its "internals", so `.mongo` becomes just a pile of unrelated stuff.

Reply:
Good call. Done in ca3abc5.